.github/workflows/publish-release-packages.yml

```diff
@@ -34,12 +34,8 @@ jobs:
         with:
           dotnet-version: 6.0.x
 
-      - name: Get version from tag
-        id: get_version
-        run: echo ::set-output name=VERSION::${GITHUB_REF/refs\/tags\//}
-
-      - name: Create CommonConstants file
-        run: scripts/update_common_constants.rb $(echo ${{ steps.get_version.outputs.VERSION }} | sed s/^v//)
+      - name: Re-generate auto-generated files
+        run: make auto-generated
```

Comment:
Given these change in the script, we no longer need to pass in the version to the script like this. This actually means that the `CUSTOM_VERSION` support which was added in #289 is not even needed any more, but we keep it for now.